packages/cli-template-contracts-hardhat/package.json

```diff
@@ -9,57 +9,67 @@
         "scripts/",
         "tasks/",
         "test/",
+        ".editorconfig",
         ".env.example",
+        ".eslintignore",
+        ".eslintrc.json",
+        ".prettierignore",
+        ".prettierrc.json",
         "hardhat.config.ts",
-        "tsconfig.json",
-        "README.md"
+        "tsconfig.json"
```

Comment:
This list is missing: `README.md`, `.yarn/`, `.yarnrc.yml`

It doesn't need: `scripts/`